SoundSwitch/Framework/NotificationManager/Notification/NotificationBanner.cs

```diff
@@ -62,51 +87,20 @@ public void NotifyMuteChanged(string microphoneName, bool newMuteState)
                 Priority = 2,
                 Image = icon,
                 Title = title,
-                Position = _bannerPositionFactory.Get(Configuration.BannerPosition)
+                Position = BannerPosition()
```

Comment:
Then this get changed to this
```suggestion
                Position = BannerPosition
```